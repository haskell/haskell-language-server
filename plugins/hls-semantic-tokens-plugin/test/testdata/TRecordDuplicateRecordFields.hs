{-# LANGUAGE DuplicateRecordFields      #-}

module TRecordDuplicateRecordFields where

data Foo = Foo { boo :: !String } 