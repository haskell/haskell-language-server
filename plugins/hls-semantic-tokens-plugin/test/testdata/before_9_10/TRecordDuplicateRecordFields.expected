5:6-9 TTypeConstructor "Foo"
5:12-15 TDataConstructor "Foo"
5:18-21 TRecordField "boo"
5:26-32 TTypeSynonym "String"
